auth_asg.tf

```diff
@@ -67,7 +67,8 @@ resource "aws_launch_configuration" "auth" {
     }
   )
   metadata_options {
-    http_tokens = "required"
+    http_endpoint = "enabled"
```

Comment:
Thanks for fixing this here. I fixed it elsewhere, but guess not in this module.

Reply:
FWIW, I chatted briefly with Walt, and we will need to merge this and whatever lives in teleport.git/examples/aws/ha-cluster . Both do the same thing, this is just not maintained - while telerpot.git copy is